homework-2/scss/style.scss

```diff
@@ -0,0 +1,1385 @@
+      text-transform: uppercase;     
+    }   
+  }
+  &-text{
+    @extend %title-section-text-mobile;
+    max-width: 316px;
+    margin-top: 15px;
+  }
+}
+.my-dots{
+  position: absolute;
+  top: 40px;
+  left: 0;
+  right: 0;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  width: 100%;
+  padding: 0;
+  list-style: none;
+  text-align: center;
+  & li{
+    position: relative;
+    display: inline-block;
+    width: 18px;
+    height: 18px;
+    margin: 0;
+    padding: 0;
+    cursor: pointer;
+    & button{
+      font-size: 0;
+      line-height: 0;
+      display: block;
+      width: 4px;
+      height: 4px;
+      padding: 4px;
+      cursor: pointer;
+      color: transparent;
+      border: 0;
+      outline: none;
+      background: #22BF95;
+      border-radius: 50%;
+      &::before{
+        position: absolute;
+        top: -5px;
+        left: -5px;
+        width: 18px;
+        height: 18px;
+        content: '';
+        text-align: center;
+        opacity: 1;
+        color: #22BF95;
+      }
+    }
+  }
+}
+.my-dots li.slick-active button:before{
+  border: 1px solid #22BF95;
+  border-radius: 50%;
+  opacity: 1;
+  color: #22BF95;
+}
+.slick-slide .slide-img{
+  max-width: 262px;
+  width: 100%;
+  margin: 20px auto 0;
+}
+.planse{
+  position: relative;
+  margin-top: -255px;
+  &-title{
+    text-align: center;
+  }
+}
+.planse-card-inner{
+  display: flex;
+  padding-left: 13px;
+}
+.planse-card{
+  position: relative;
+  max-width: 260px;
+  width: 100%;
+  margin: 30px 13px 0 0;
+  padding: 35px 23px 53px;
+  background: #FFFFFF;
+  box-shadow: $button-box-shadow;
+  border-radius: 5px; 
+  transition: transform 0.3s ease;
+  transform: scaleY(1);
+  .card-title{
+    color: $color-title-card;
+    font-family: $font-title, sans-serif;
+    font-size: 12px;
+    line-height: 24px;
+    letter-spacing: 2px;
+    text-align: center;
+    letter-spacing: 2px;
+    text-transform: uppercase;
+  }
+  .btn-request{
+    max-width: 208px;
+    width: 100%;
+    font-size: 12px;
+    line-height: 24px;
+    text-align: center;
+    letter-spacing: 2px;
+    text-transform: uppercase;
+    color: $color-button;
+    background: $button-bg-light;
+    border: 1px solid #22BF95;
+    box-shadow: 0px 6px 12px rgba(127, 127, 127, 0.3);
+    border-radius: 20px;
+    padding: 8px 0;      
+  }
+  &-list{
+    margin-top: 30px;
+  }
+}
+.slider-logo-single{
+  .slick-slide .single-img{
+    max-width: 235px;
+    width: 100%;
+    margin: 0 auto;
+  }
+  .slick-disabled{
+    opacity: 0.5;
+  }
+}
+.slider-logo-list{
+  margin-top: 40px;
+  .img-logo{
+    max-width: 51px;
+    width: 100%;
+  }
+}
+.slick-dots{
+  display: flex;
+  width: 100%;
+  li{    
+    flex: 1;
+    background: #E5E5E5;
+    height: 5px;
+
+    &.slick-active{
+      background: #22bf95;
+    }
+    button{
+      visibility: hidden;
+    }
+  }
+}
+.slider-logo{
+  padding: 100px 0; 
+}
+.description{
```

Comment:
You need to be consistent when creating class names - when you write `.slider-logo` - it is pretty clear, what it means.

Here `.description` - I have no idea where it is on page.

Reply:
done